### PR TITLE
Update sandbox and fix lumos color

### PR DESCRIPTION
**Pull request by NotNestor**
## Description
* Fix Lumos selected tab colour
* Replace navigation in sandbox with mdtabs
* Fix issue in md-tabs not preventing default handling of keypresses
* Update params for grabber
* Add gradient divider

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/combobox/tokens/md-combobox-tokens.js [hunk 1]
```diff
@@ -74,8 +74,8 @@ const combobox = {
         dark: colors.blue[40].name
       },
       "select-bg-color": {
-        light: colors.white[60].name,
-        dark: colors.gray[90].name
+        light: colors.blue[50].name,
+        dark: colors.blue[40].name
       },
```
Comment (NotNestor) [inline web-components/src/components/combobox/tokens/md-combobox-tokens.js:79]: Fixes bug CX-15442

File: web-components/package.json [hunk 2]
```diff
@@ -114,8 +114,7 @@
     "clean-webpack-plugin": "^4.0.0",
     "copy-webpack-plugin": "^5.1.1",
     "css-loader": "^3.4.2",
-    "cssstats": "^3.4.1",
-    "elix": "^12.1.4",
+    "cssstats": "^3.4.1",    
     "eslint": "^8.56.0",
```
Comment (NotNestor) [inline web-components/package.json:118]: remove elix and use the toolkit components in the sandbox

File: web-components/src/components/grabber/tokens/lm-grabber-tokens.js [hunk 3]
```diff
@@ -26,12 +26,13 @@ const grabber = {
...
   width: {
-    light: "14px",
-    dark: "14px"
+    common: "14px"
   },
   "border-radius": {
-    light: "16px",
-    dark: "16px"
+    common: "16px"
+  },
```
Comment (NotNestor) [inline web-components/src/components/grabber/tokens/lm-grabber-tokens.js:33]: update to new format implemented by @Jason20020

File: web-components/src/components/tabs/Tabs.ts [hunk 4]
```diff
@@ -860,6 +861,7 @@ export namespace Tabs {
           } else if (tab && !tab.disabled) {
             const idx = this.tabsIdxHash[this.getNormalizedTabId(tab.id)];
             if (idx !== -1) {
+              event.preventDefault();
               this.updateSelectedTab(idx);
             }
```
Comment (NotNestor) [inline web-components/src/components/tabs/Tabs.ts:866]: fix issue where space to active also scrolled scroll view as prevent default wan't called

File: web-components/src/components/tabs/Tabs.ts [hunk 5]
```diff
@@ -844,6 +844,7 @@ export namespace Tabs {
           break;
         }
         case Key.Enter:
+        case Key.NumpadEnter:
         case Key.Space: {
```
Comment (NotNestor) [inline web-components/src/components/tabs/Tabs.ts:848]: numpad enter should also active. These should probably be added to a utility function like isActionKeypress

File: web-components/src/components/theme/Theme.ts [hunk 6]
```diff
@@ -361,8 +361,8 @@ export namespace Theme {
             }
           </style>
           <slot></slot>
-          <div class="md-tooltip" virtual-global-popper></div>
-          <div virtual-global-reference aria-describedby="tooltip"></div>
+          <div class="md-tooltip" data-virtual-global-popper></div>
+          <div data-virtual-global-reference aria-describedby="tooltip"></div>
         </div>
```
Comment (NotNestor) [inline web-components/src/components/theme/Theme.ts:366]: update to recommended

**Review by AlanSpillane — APPROVED** [review]

**Pull request merged**